indexer/config/config.go

```diff
@@ -11,6 +11,12 @@ import (
 	geth_log "github.com/ethereum/go-ethereum/log"
 )
 
+const (
+	// default to 5 seconds
+	defaultLoopInterval     = 5000
```

Comment:
We should have a better default here, and instead override the indexer config in `e2e_tests/setup.go` to something like 1 or 2 seconds.

Just spitballing: with a ~15s second block time, how about ~3 minutes so we elapse at least 10 L1 blocks?

Reply:
And probably a tighter loop interval on L2 since the block time is ~2s. Maybe like 1 minute on L2?

Reply:
Hmm I'm curious as to why we'd want a longer interval that allows us to not know about blocks for awhile. Would it not make more sense to do polling significantly (i.e, in-between new blocks) more frequently so that we can capture and process new block events as they happen in almost real-time?

Reply:
hmm the primary tradeoff is # RPCs vs efficiency via batching. But i can see this being pretty lightweight enough to run on each block.

ffti and lets maybe re-evaluate how much overhead the indexer running on this interval is adding to the replica it's connected with